datasrc/languages/bulgarian.txt

```diff
@@ -2093,3 +2093,6 @@ Your stake got returned: +%lld money.
 
 You got +%lld money from others leaving the game.
 == 
+
+available
+== налична
```

Comment:
There is still one translation in each language file which shouldn't be there :D

Reply:
<img width="227" height="85" alt="grafik" src="https://github.com/user-attachments/assets/ac026c5d-7440-48c5-b32e-bd5908514532" />

Check *each* file